Content.Client/Mapping/MappingPrototype.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Client.Mapping;
+
+public sealed class MappingPrototype
+{
```

Comment:
Doc comment I beg.

Reply:
Done in [`a6c0bb4` (#23427)](https://github.com/space-wizards/space-station-14/pull/23427/commits/a6c0bb468d5457b3ee742c1b8363940bb1660cd0)